Documentation/cockroachdb-cluster-crd.md

```diff
@@ -50,6 +56,7 @@ Under the `scope` field, a `StorageScopeSpec` can be specified to influence the
 These properties are currently supported:
 
 * `nodeCount`: The number of CockroachDB instances to create.  Some of these instances may be scheduled on the same nodes, but exactly this many instances will be created and included in the cluster.
+* `volumeClaimTemplates`: .
```

Comment:
I assume you're going to fill this out with details? :)

Reply:
Fixed in both docs. PTAL